scripts/core/virtual_memory.yml

```diff
@@ -303,3 +303,45 @@ params:
     - type: $x_physical_mem_handle_t
       name: hPhysicalMem
       desc: "[in][release] handle of the physical memory object to release."
+
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Physical memory range info queries."
+class: $xPhysicalMem
+name: $x_physical_mem_info_t
+typed_etors: True
+etors:
+# properties and size too
```

Comment:
Is this a todo?

Reply:
oh yes it is, urVirtualMemoryCreate takes a context, device, size and properties too, so these should be added to GetInfo too. I'll add a new commit to include these

Reply:
Added the additional properties.